spec/features/admin/admin_manages_items_spec.rb

```diff
@@ -86,15 +86,15 @@
     # expect(page).to have_css("img[alt='Cookie monster']")
   end
 
-  xit "can retire an item" do
+  it "can retire an item" do
     create_admin_user
     item = create(:item)
     item.categories << create(:category)
     visit edit_menu_item_path(item)
-    page.execute_script("$('#item_status').val('retired')")
+    select 'retired', from: 'item_status'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.